### PR TITLE
add rtl styling for dropdown component

### PR DESCRIPTION
**Pull request by rodleviton**
dropdown

### CODE REVIEW

File: components/Dropdown/DropdownMenu.js [hunk 1]
```diff
@@ -31,21 +31,23 @@ export default class DropdownMenu extends React.Component<DropdownMenuProps> {
...
     const rect = menu.getBoundingClientRect();
+
     if (pos.bottom > innerHeight - rect.height) {
       menu.style.bottom = '24px';
       menu.style.top = 'auto';
     } else {
       menu.style.top = '24px';
       menu.style.bottom = 'auto';
     }
-    menu.style.right = '0px';
```
Comment (sebpearce) [inline components/Dropdown/DropdownMenu.js]: What did removing this do? Assuming it was there for a reason, wouldn't it need the equivalent in RTL as `left`? Or was it redundant?

Comment (rodleviton) [reply components/Dropdown/DropdownMenu.js]: I needed to control rtl/ltr and it was easier to remove this style here and add it to the css.

https://github.com/cultureamp/cultureamp-style-guide/pull/144/files#diff-f361c10470ae7ab5237d91c2c269b606R85

Comment (sebpearce) [reply components/Dropdown/DropdownMenu.js]: Gotcha 👍

**Pull request merged**